vocabularies/Common.json

```diff
@@ -1288,6 +1288,138 @@
+          }
+        ],
+        "$ReturnType": {
+          "@Core.IsURL": true,
+          "@Core.Description": "URL (relative to the service URL) for retrieving the recommendations (which are computed asynchronously)",
+          "@Core.Example": {
+            "@odata.type": "https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Core.V1.xml#Core.PrimitiveExampleValue",
+            "Description": "Relative URL for invoking a function import of the [retrieval function](#Template_RecommendationsRetrievalFunction)",
+            "Value": "RetrieveRecommendations?ID=af43c2"
+          }
+        }
+      }
+    ],
+    "Template_RecommendationsRetrievalFunction": [
+      {
+        "$Kind": "Function",
+        "@Common.Experimental": true,
+        "@Core.Description": "Template for functions that retrieve AI-based recommendations for the bound entity and its related entities",
+        "@Core.LongDescription": "The template function itself cannot be invoked.",
+        "$Parameter": [
+          {
+            "$Name": "ID",
+            "@Core.Description": "ID that occurs in the URL returned by a prior invocation of the [recommendations function](#Template_RecommendationsFunction)"
+          }
+        ],
+        "$ReturnType": {
+          "$Collection": true,
+          "$Type": "Common.PropertyRecommendationType",
+          "@Core.Description": "A collection of recommendations per targeted property"
+        }
+      }
+    ],
+    "Template_RecommendationsFunction2": [
+      {
+        "$Kind": "Function",
+        "$IsBound": true,
+        "@Common.Experimental": true,
+        "@Core.Description": "Variant of [`Template_RecommendationsFunction`](#Template_RecommendationsFunction)",
+        "@Core.LongDescription": "The function returns the recommendations but can be invoked with `Prefer: respond-async`.\n\nThe template function itself cannot be invoked.",
+        "$Parameter": [
+          {
+            "$Name": "Entity",
+            "$Type": "Edm.EntityType",
+            "@Core.Description": "See parameter `Entity` in [`Template_RecommendationsFunction`](#Template_RecommendationsFunction)"
+          },
+          {
+            "$Name": "Targets",
+            "$Collection": true,
+            "@Core.IsURL": true,
+            "@Core.Description": "See parameter `Targets` in [`Template_RecommendationsFunction`](#Template_RecommendationsFunction)"
+          }
+        ],
+        "$ReturnType": {
+          "$Collection": true,
+          "$Type": "Common.PropertyRecommendationType",
+          "@Core.Description": "See return type in [`Template_RecommendationsRetrievalFunction`](#Template_RecommendationsRetrievalFunction)"
+        }
+      }
+    ],
+    "PropertyRecommendationType": {
+      "$Kind": "ComplexType",
+      "@Common.Experimental": true,
+      "target": {
```

Comment:
Should this probably be `Target` (capital `T`)? I think this would be more consistent with the rest of the vocabulary. Maybe the other properties of this type and `AlternativeRecommendationType`, too.

Reply:
To be discussed in Albatross meeting.

Reply:
Similarity with messages - keep lower case.